### PR TITLE
Update inference.py

### PR DESCRIPTION
**Pull request by AresGao**
fix inference bug, the default keep_ratio is True...But SSD should be false

### CODE REVIEW

**Comment by tjsongzw** [comment]
`cfg.data.test.get('resize_keep_ratio', True)` would be better, it will keep default behavior on other config files.

**Comment by AresGao** [comment]
> `cfg.data.test.get('resize_keep_ratio', True)` would be better, it will keep default behavior on other config files.

Yes, u are right....
But why it checks failed..

**Comment by tjsongzw** [comment]
seems it didn't pass flake8 checking

File: mmdet/apis/inference.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
 def _prepare_data(img, img_transform, cfg, device):
     ori_shape = img.shape
     img, img_shape, pad_shape, scale_factor = img_transform(
-        img, scale=cfg.data.test.img_scale)
+        img, scale=cfg.data.test.img_scale,
```
Comment (hellock) [inline mmdet/apis/inference.py]: You have a trailing whitespace here, which is unexpected and fails the flake8 linting.

If you use yapf to auto-format the code, it should look like this.
```python
img, img_shape, pad_shape, scale_factor = img_transform(
    img,
    scale=cfg.data.test.img_scale,
    keep_ratio=cfg.data.test.get('resize_keep_ratio', True))
```

**Pull request merged**